business_requirement_deliverable/models/business.py

```diff
@@ -244,6 +309,68 @@ class BusinessRequirement(models.Model):
         readonly=True,
         compute='_compute_get_currency',
     )
+    dl_count = fields.Integer('DL Count', compute='_compute_dl_count')
+    rl_count = fields.Integer('DL Count', compute='_compute_rl_count')
+
+    @api.multi
+    def _compute_dl_count(self):
+        self.dl_count = self.env['business.requirement.deliverable'].\
+            search_count([('business_requirement_id', '=', self.id)])
+
+    @api.multi
+    def _compute_rl_count(self):
+        for r in self:
+            r.rl_count = self.env['business.requirement.resource'
```

Comment:
Please use another indentation scheme that is not so ugly. Example:
```
r.rl_count = self.env[
    'business.requirement.resource'
].search_count(
    [('business_requirement_id', '=', r.id)
)
```
Anyway this can be replace by a one2many field, and use `len()` over that field.